### PR TITLE
column: fix zero value for binary type

### PR DESCRIPTION
**Pull request by coocood**
Zero value for BINARY type should be byte slice with length.

### CODE REVIEW

**Comment by coocood** [comment]
@tiancaiamao @zimulala @zz-jason PTAL

**Comment by zimulala** [comment]
LGTM

File: table/column.go [hunk 1]
```diff
@@ -370,7 +371,13 @@ func GetZeroValue(col *model.ColumnInfo) types.Datum {
...
 	case mysql.TypeNewDecimal:
 		d.SetMysqlDecimal(new(types.MyDecimal))
-	case mysql.TypeString, mysql.TypeVarString, mysql.TypeVarchar:
+	case mysql.TypeString:
+		if col.Flen > 0 && col.Charset == charset.CharsetBin {
+			d.SetBytes(make([]byte, col.Flen))
+		} else {
+			d.SetString("")
+		}
+	case mysql.TypeVarString, mysql.TypeVarchar:
```
Comment (zz-jason) [inline table/column.go]: should we consider `TypeVarString` ?

Comment (coocood) [reply table/column.go]: No, `VarString` is not fixed length type, does not padding to the length.

**Pull request merged**